NuGet.Services.Dashboard.Operations/Infrastructure/ConfigurationProcessor.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Configuration;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace NuGet
+{
+    public class ConfigurationProcessor
+    {
+        private ISecretReaderFactory _secretReaderFactory;
+
+        public ConfigurationProcessor(ISecretReaderFactory secretReaderFactory)
+        {
+            _secretReaderFactory = secretReaderFactory;
+        }
+
+        public void InjectSecretsIntoConfiguration()
```

Comment:
Agree with Maarten. Please use ConfigurationManager.AppSettings as a parameter to SecretReaderFactory as well

---
In reply to: [72012691](https://github.com/NuGet/NuGet.Services.Dashboard/pull/187#discussion_r72012691) [](ancestors = 72012691)